### ISSUE TITLE
CW UI suggestion: Cloud/Bucket icons to reflect remote/local database location  

### ISSUE THREAD

**Issue by johs** [ermouth/cloudwall#20]
One of the things that I think would help the conceptual understanding of CW would be to establish a better understanding of what is in the cloud and what is local.
The fact that CW is a web based system created by itself, presenting a offline and local environment with copies of databases that could live in various places is the hard thing to grasp for new users.
This get really complicated when you get to _D with the ability to push content to databases that has nothing to do with where the source data resides. "What is where and where is what?" becomes really confusing unless the user has a very straight system. If the IDE should hold several projects that each consists of several databases, it starts to get messy and you wonder "what is this thing".

One idea would be to present the left pane as the connection to the cloud. Since the databases can belong to various clouds, the cloud icon for the bucket indicate that this is connection to several "clouds".

The same icon in the main pane could have a bucket icon to signify that you are now dealing with the local copy. Once you sync, the cloud and the bucket are the same, so that is the connection between two different things (ref suggestion to move sync icon).

The left pain could be cleaned up to deal with the cloud connection only, especially the presentation of the System DB is confusing when it comes to system apps also being in available together with apps in each DB. Here I think a better idea is to not talk about ther system as a database at all, just the system.

To sum up, my suggestion is:
1. Present external data as Cloud to accentuate that these can reside in different servers/clouds
2. Present local data as Bucket
3. Keep sync button between ther two
4. Move system DB to e.g. the CW logo and call it system settings or just system, not to be confused with data
5. Change "add DB" to "Connect new DB" and keep the left pane as a single-purpose list of assets online loaded to the IDE to work with.

To:

![screen shot 2017-01-28 at 06 25 46](https://cloud.githubusercontent.com/assets/48370/22394748/ef18c35c-e528-11e6-9f27-d483dbb49aaa.jpg)

(Existing):
![screen shot 2017-01-28](https://cloud.githubusercontent.com/assets/48370/22394758/41171ffa-e529-11e6-8658-245a50fab16d.jpg)
